pkg/db/db.go

```diff
@@ -20,14 +20,13 @@ var DB *gorm.DB
 
 // GetUrl Get database config and return url
 func GetUrl() string {
-	conf := config.Get()
```

Comment:
so this isn't possible to do: 
conf := config.Get().Database 

I think I don't get why we are doing this still.

Reply:
Everything else looks good, just asking this more for my own understanding.